packages/dds/merge-tree/src/test/client.conflictFarm.spec.ts

```diff
@@ -23,16 +24,22 @@ interface IConflictFarmConfig extends IMergeTreeOperationRunnerConfig {
 	clients: IConfigRange;
 }
 
-const allOperations: TestOperation[] = [removeRange, annotateRange, insertAtRefPos];
+const allOperations: TestOperation[] = [
+	removeRange,
+	annotateRange,
+	insertAtRefPos,
+	obliterateRange,
+];
 
 export const debugOptions: IConflictFarmConfig = {
-	minLength: { min: 1, max: 1 },
-	clients: { min: 3, max: 3 },
-	opsPerRoundRange: { min: 1, max: 100 },
-	rounds: 1000,
+	minLength: { min: 1, max: 512 },
+	clients: { min: 1, max: 8 },
+	opsPerRoundRange: { min: 1, max: 128 },
+	rounds: 8,
```

Comment:
This is a rather dramatic decrease in `rounds`. What's the reasoning behind this? Were the tests taking too long?

Reply:
Ah, this is just the debug options, which aren't actually referenced anywhere in main. I'm not sure why we have four separate options objects here. Seems like a lot of repetition could be eliminated by doing something like `export const debugOptions: IConflictFarmConfig = { ...defaultOptions, incrementalLog: true }`

Reply:
it's just for ease of use. i frequently copy the default options here, and then tweak them until i hit the case i'm trying to hit.